### ISSUE TITLE
add a label showing the inspiratory time in settings ui

### ISSUE THREAD

**Issue by carmelopellegrino** [MechanicalVentilatorMilano/gui#190]
In **settings ui, PCV tab only**, add a label that changes the value of the inspiratory time according to the values specified by the user in the `RR` and `I:E` input fields, called in the code `spinbox_rr` and `spinBox_insp_expir_ratio` respectively.

The formula to calculate this time is:
```
I = (60.0 seconds) / (spinbox_rr * (1+spinBox_insp_expir_ratio))
```

The displayed `I` value must change at any change of both `spinBox_insp_expir_ratio` and `spinbox_rr` values.

The upcoming PR must be merged in `issue-185` branch rather than in `master`.

**Comment by marcodeltutto** [comment]
Hi Carmelo, looks good to me but when I run this on my Pi I can't see the `I:` label that should be next to `I:E`. Maybe we can split the label that contains both `I:E` and `I:`? This is how it looks:
![IMG_3936](https://user-images.githubusercontent.com/17006198/80123142-1b4d0180-8554-11ea-9741-af3df5ac8e5d.jpeg)

**Comment by carmelopellegrino** [comment]
solved in PR #191
